source/slang/core.meta.slang

```diff
@@ -2561,6 +2561,9 @@ attribute_syntax [nonmutating] : NonmutatingAttribute;
 __attributeTarget(FunctionDeclBase)
 attribute_syntax [constref] : ConstRefAttribute;
 
+__attributeTarget(FunctionDeclBase)
+attribute_syntax [__ref] : RefAttribute;
```

Comment:
Because this new attribute is only for `this`, how about a name like `__ref_this` to be more explicit.